deploy/nginx/nginx.conf

```diff
@@ -52,7 +52,65 @@ http {
         rewrite ^/developer/index.html /install-with-other/docker permanent;
         rewrite ^/getting-started/px-licensing.html /reference/release-notes/px-licensing permanent;
         rewrite ^/scheduler/kubernetes/ark.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/disaster-recovery permanent;
-        rewrite ^/scheduler/kubernetes/(.*) /portworx-install-with-kubernetes permanent;
+        ark.md
+        rewrite ^/scheduler/kubernetes/cassandra-k8s.html /portworx-install-with-kubernetes/application-install-with-kubernetes/cassandra permanent;
+        rewrite ^/scheduler/kubernetes/csi.html /portworx-install-with-kubernetes/storage-operations/csi permanent;
+        rewrite ^/scheduler/kubernetes/docker-to-oci.html /install-with-other/docker/standalone permanent;
```

Comment:
change this (/scheduler/kubernetes/docker-to-oci.html) to /portworx-install-with-kubernetes